Source/WebKit/WebProcess/GPU/media/RemoteAudioDestinationProxy.cpp

```diff
@@ -108,7 +109,12 @@ IPC::Connection* RemoteAudioDestinationProxy::connection()
         m_gpuProcessConnection = gpuProcessConnection;
         gpuProcessConnection->addClient(*this);
         m_destinationID = RemoteAudioDestinationIdentifier::generate();
-        gpuProcessConnection->connection().send(Messages::RemoteAudioDestinationManager::CreateAudioDestination(m_destinationID, m_inputDeviceId, m_numberOfInputChannels, m_outputBus->numberOfChannels(), sampleRate(), m_remoteSampleRate, m_renderSemaphore), 0);
+
+        m_lastFrameCount = 0;
```

Comment:
I'm not sure if line 113 (initialization to 0) is necessary. I.e., is there a situation where we re-create the connection?